helm/defectdojo/values.yaml

```diff
@@ -197,6 +197,8 @@ django:
   replicas: 1
   tolerations: []
   uwsgi:
+    env:
```

Comment:
As env is in use for other components, not only for  the  django uwsgi itself, this is not natural position for given config

Reply:
Done ! it seems odd to have it all the way down in the values.yaml but I understand yes, I added a comment on the default value to explain why you likely would change the value